src/utils/pino.ts

```diff
@@ -56,7 +56,7 @@ const prettyPrint = {
 
       const probeMsg = `${log.iteration} id:${log.id} ${log.responseCode} ${log.url} ${log.responseTime}ms`
 
-      if (log.notification?.flag) {
+      if (log.notification?.flag.length > 2) {
```

Comment:
why checking the flag string to be longer than 2?

Reply:
oh good catch.. was a trial of something thats nologer relevant. removed.